app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,58 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      erb :'/tweets/tweets'
+    else redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    redirect '/tweets/new' if params[:content].empty?
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.

```
@tweet = Tweet.new(content: params[:content], user_id: current_user.id)
if @tweet.save
  erb :'/tweets/show_tweet'
else
  redirect '/tweets/new'
end
```